src/style.css

```diff
@@ -1 +1,24 @@
-/* add styles here */
+* {
+  margin: 0;
+  box-sizing: border-box;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  background-image: url(./images/star.svg);
+  background-repeat: no-repeat;
+  height: 16px;
+  width: 16px;
+  margin-left: 3px;
+}
+
+.stars--1 .stars__star:nth-child(1),
+.stars--2 .stars__star:nth-child(-n + 2),
+.stars--3 .stars__star:nth-child(-n + 3),
+.stars--4 .stars__star:nth-child(-n + 4),
+.stars--5 .stars__star:nth-child(1n) {
```

Comment:
The selector `.stars--5 .stars__star:nth-child(1n)` is equivalent to selecting all children. It might be clearer to use `.stars--5 .stars__star` to indicate that all stars should have the active background image.